src/freebsd/btop_collect.cpp

```diff
@@ -1186,9 +1191,17 @@ namespace Proc {
 				}
 			}
 
-			//? Clear dead processes from current_procs
-			auto eraser = rng::remove_if(current_procs, [&](const auto &element) { return not v_contains(found, element.pid); });
-			current_procs.erase(eraser.begin(), eraser.end());
+			//? Clear dead processes from current_procs if not paused
+			if (not pause_proc_list) {
+				auto eraser = rng::remove_if(current_procs, [&](const auto &element) { return not v_contains(found, element.pid); });
```

Comment:
```suggestion
				auto eraser = rng::remove_if(current_procs, [&](const auto& element) { return not v_contains(found, element.pid); });
```

Left aligned reference symbols, please.

Reply:
> Left aligned reference symbols, please.

I will make this change. that is a line I didn't personally write though. I just wrapped a if statement around it. but I will still do that change